### PR TITLE
v2.0

### PR DESCRIPTION
**Pull request by stevenwilkin**

### CODE REVIEW

File: app/domain/output_document.rb [hunk 1]
```diff
@@ -5,7 +5,9 @@ class OutputDocument
 
   delegate :id, :income_in_retirement, :continue_working, :unsure,
```
Comment (andrewgarner) [inline app/domain/output_document.rb]: We only need to delegate attributes that are needed for the output document. That means we can skip the circumstances and possibly pot size and income fields?

File: features/preview_record_of_guidance.feature [hunk 2]
```diff
@@ -3,50 +3,17 @@ Feature: Record of guidance preview
...
   Then the sections that the preview includes should be (in order):
-    | introduction             |
-    | pension pot              |
+    | covering letter          |
+    | getting started          |
     | options overview         |
     | detail about each option |
-    | other information        |
-
-Scenario Outline: Guidance is tailored based on applicable circumstances
```
Comment (andrewgarner) [inline features/preview_record_of_guidance.feature]: We should have an acceptance test that proves that the applicable circumstances are captured in the database for analytical purposes.

```gherkin
Given "<circumstance>" applies to the customer
When I create their record of guidance
Then information about "<circumstance>" should be recorded
```

File: features/preview_record_of_guidance.feature [hunk 3]
```diff
@@ -3,50 +3,17 @@ Feature: Record of guidance preview
...
-  Given one or more of the predefined circumstances applies to the customer
-  When I preview their record of guidance
-  Then the sections that the preview includes should be (in order):
-    | introduction                          |
-    | pension pot                           |
-    | options overview                      |
-    | detail about applicable circumstances |
-    | other information                     |
-
 Scenario: Generic record of guidance
```
Comment (andrewgarner) [inline features/preview_record_of_guidance.feature]: We should referred to them as 'standard' records not 'generic' records.

File: app/domain/output_document.rb (hunk 1 above)
Comment (stevenwilkin) [inline app/domain/output_document.rb]: Good catch, I'm sure I'd removed them before but then I went crazy with the rebasing. I've changed the delegated attributes to just the id and the supplementary topics and everything's green.

File: features/preview_record_of_guidance.feature (hunk 2 above)
Comment (stevenwilkin) [inline features/preview_record_of_guidance.feature]: Makes sense!

File: features/preview_record_of_guidance.feature (hunk 3 above)
Comment (stevenwilkin) [inline features/preview_record_of_guidance.feature]: I've updated the scenario title

**Comment by stevenwilkin** [comment]
Updated to take feedback into consideration.

Have included 1c874034147c0ee644703a712d40ff431d4ff17c with an acceptance test for the individual circumstances which are no longer part of the summary document but which we still want to store in the db.

File: features/record_of_guidance.feature [hunk 4]
```diff
@@ -0,0 +1,19 @@
+Feature: Record of guidance
```
Comment (andrewgarner) [inline features/record_of_guidance.feature]: :+1: 

Can we also include a scenario(s) to cover the additional information we're collecting but not including in the document. Off the top of my head I can think of:

- `income_in_retirement`
- `value_of_pension_pots`
- `upper_value_of_pension_pots`
- `value_of_pension_pots_is_approximate`

Comment (stevenwilkin) [inline features/record_of_guidance.feature]: Good call, these can probably be covered in the one scenerio

Comment (stevenwilkin) [inline features/record_of_guidance.feature]: Updated

**Comment by andrewgarner** [comment]
:+1:

**Pull request merged**